[2023-12-10 19:33:12,281] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-10-25T00:00:00+00:00 [queued]>
[2023-12-10 19:33:12,504] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-10-25T00:00:00+00:00 [queued]>
[2023-12-10 19:33:12,514] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:33:12,518] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2023-12-10 19:33:12,520] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-12-10 19:33:12,758] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): segunda_tarea> on 2022-10-25 00:00:00+00:00
[2023-12-10 19:33:12,794] {standard_task_runner.py:52} INFO - Started process 1057 to run task
[2023-12-10 19:33:12,803] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'mi_primer_dag_v3', 'segunda_tarea', 'scheduled__2022-10-25T00:00:00+00:00', '--job-id', '259', '--raw', '--subdir', 'DAGS_FOLDER/primer_dag_v3.py', '--cfg-path', '/tmp/tmp22qhzl3p', '--error-file', '/tmp/tmp7f3rf15b']
[2023-12-10 19:33:12,845] {standard_task_runner.py:80} INFO - Job 259: Subtask segunda_tarea
[2023-12-10 19:33:13,448] {task_command.py:371} INFO - Running <TaskInstance: mi_primer_dag_v3.segunda_tarea scheduled__2022-10-25T00:00:00+00:00 [running]> on host 42828894a739
[2023-12-10 19:33:14,538] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=DavidBU
AIRFLOW_CTX_DAG_ID=mi_primer_dag_v3
AIRFLOW_CTX_TASK_ID=segunda_tarea
AIRFLOW_CTX_EXECUTION_DATE=2022-10-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-25T00:00:00+00:00
[2023-12-10 19:33:14,547] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2023-12-10 19:33:14,551] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo hola, soy la tarea 2 y sere corrida luego de la Tarea 1']
[2023-12-10 19:33:14,635] {subprocess.py:85} INFO - Output:
[2023-12-10 19:33:14,651] {subprocess.py:92} INFO - hola, soy la tarea 2 y sere corrida luego de la Tarea 1
[2023-12-10 19:33:14,669] {subprocess.py:96} INFO - Command exited with return code 0
[2023-12-10 19:33:14,949] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=mi_primer_dag_v3, task_id=segunda_tarea, execution_date=20221025T000000, start_date=20231210T193312, end_date=20231210T193314
[2023-12-10 19:33:15,026] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-12-10 19:33:15,283] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
